catalog/core/catalog-core-localstorageprovider/src/main/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProvider.java

```diff
@@ -178,6 +179,22 @@ public UpdateStorageResponse update(UpdateStorageRequest updateRequest)
             }
         }
 
```

Comment:
This is fixing a bug for updated content items. Previously we did not update the resourceUri or resourceSize, this ensures they get set when updated.